app/components/general/components/bc-table/macro.njk

```diff
@@ -7,10 +7,19 @@
 {% macro bcTable(params) %}
   <div data-test-id="table">
     {% if params.columnInfo %}
-      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 nhsuk-u-padding-bottom-3 bc-t-border-bottom--grey-3">
+      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 bc-t-border-bottom--grey-3">
         {% for column in params.columnInfo %}
           <div data-test-id="column-heading-{{ loop.index0 }}" class="nhsuk-u-padding-1 nhsuk-u-font-weight-bold {{ params.columnClass }}">{{ column.data }}</div>
         {% endfor %}
+        {% for column in params.columnInfo %}
```

Comment:
we got the for loop in there twice? can we just add the if clause to the one above?

Reply:
makes it go weird when there's not the extra for loop in 
<img width="934" alt="image" src="https://user-images.githubusercontent.com/30870498/84408705-60acb780-ac04-11ea-9459-3870788b106a.png">

Reply:
it is effectively another row in the headings

Reply:
mmmm doesn't seem right the one for the normal data row doesn't do that